downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -2,41 +2,928 @@
+time:Civil time = {
+    year: 2021,
+    month: 4,
+    day: 12,
+    hour: 23,
+    minute: 20,
+    second: 50.52,
+    timeAbbrev: "Asia/Colombo",
+    utcOffset: zoneOffset
+};
+task:Error|task:JobId id = task:scheduleOneTimeJob(new MyJob(), time);
+```
+
+- Schedules the recurring Ballerina job according to the given duration.
+```ballerina
+import ballerina/task;
+
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+task:Error|task:JobId id = task:scheduleJobRecurByFrequency(new MyJob(), 1);
+```
+
+- Unschedules the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:unscheduleJob(id);
+```
+
+- Pauses all the jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:pauseAllJobs();
+```
+
+- Resumes all the jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:resumeAllJobs();
+```
+
+- Pauses the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:pauseJob(id);
+```
+
+- Resumes the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:resumeJob(id);
+```
+
+- Gets all the running jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:getRunningJobs();
+```
+
+##### Time Package Updates
+
+Revamped the entire time package as follows:
+
+- Introduced the `time:Utc` record to represent the UTC timestamp.
+- Introduced the `time:Civil` record to represent the localized time.
+- Added necessary APIs to do time generation, manipulations, and conversions.
+
+Steps for migration from the previous version to the current version are listed [in this issue](https://github.com/ballerina-platform/ballerina-standard-library/issues/1079).
+
+##### Cache Package Updates
+
+- Introduced the new `EvictionPolicy` configuration to set the eviction policy in the `CacheConfig` record.
+
+The `EvictionPolicy` record has been introduced with the option `LRU` as the module only supports the LRU eviction policy to evict the cache data when the cache is full.
+
+- Removed the `AbstractEvictionPolicy` object type.
+
+This object type had the common APIs for the cache eviction functionalities to implement a custom eviction policy. It has been removed with the introduction of the above configuration.
+
+##### New `xmldata` Package
+
+A new module is added to convert data in XML format to JSON format and vice-versa.
+
+- Converts a JSON object to an XML representation.
+```ballerina
+import ballerina/xmldata;
+
+json data = {
+    name: "John",
+    age: 30
+};
+xml|xmldata:Error x = xmldata:fromJson(data);
+```
+
+- Converts an XML value to its JSON representation.
+```ballerina
+import ballerina/xmldata;
+
+json|xmldata:Error j = xmldata:toJson(xml `foo`);
+```
+
+##### Remove `jsonutils`, `xmlutils`, `runtime`, and `reflect` Packages
+
+The `jsonutils`, `xmlutils`, `runtime`, and `reflect` packages are removed from Standard Libraries.
```

Comment:
Are removed or were removed? 🤔 And should be "Standard Library" right?

Reply:
Yeah, it should be "were removed". Updated in 476974e

Reply:
Refer https://github.com/ballerina-platform/ballerina-dev-website/pull/1995#discussion_r598648159